src/glmtools.jl

```diff
@@ -11,6 +11,27 @@ type LogLink <: Link end
 type ProbitLink <: Link end
 type SqrtLink <: Link end
 
+"""
+    linkfun(::Link, μ)
+
+Link function mapping μ to η (the linear predictor).    
+"""
+function linkfun end
+
+"""
+    linkinv(::Link, η)
+
+Inverse Link function mapping mu to eta, the linear predictor (CauchitLink response model).
```

Comment:
For consistency with above, `mapping μ to η` (the linear predictor). AFAIK `CauchitLink|` is only one possible link, so just remove this part.